instrumentation/Makefile

```diff
@@ -33,8 +33,10 @@ test: tests
 dist:
 	@mkdir -p dist
 	docker buildx build --platform linux/$(ARCH) -o type=image,name=libsplunk-builder:$(ARCH),push=false .
-	docker run --platform linux/$(ARCH) --rm -v $(CURDIR)/dist:/libsplunk/so libsplunk-builder:$(ARCH) make test all
-	cp $(CURDIR)/dist/libsplunk.so $(CURDIR)/dist/libsplunk_$(ARCH).so
+	docker run -d --platform linux/$(ARCH) --name libsplunk-builder libsplunk-builder:$(ARCH) sleep inf
+	docker exec libsplunk-builder make test all
+	docker cp libsplunk-builder:/libsplunk/so/libsplunk.so dist/libsplunk_$(ARCH).so
+	docker rm -f libsplunk-builder
```

Comment:
Need to adjust since we can't mount writable volumes in gitlab runner containers.